superset-frontend/src/views/CRUD/annotation/AnnotationModal.tsx

```diff
@@ -114,6 +114,16 @@ const AnnotationModal: FunctionComponent<AnnotationModalProps> = ({
   // Functions
   const hide = () => {
     setIsHidden(true);
+
+    // Reset annotation
+    setCurrentAnnotation({
+      short_descr: '',
```

Comment:
looks like this object is used twice, let's define it as a constant at the top of the file and use it as needed throughout this function

Reply:
Or create a short helper function `resetAnnotation()`.